docs/EmbeddedSwift/UserManual.md

```diff
@@ -98,6 +98,32 @@ Segment __TEXT: 16384
 ...
 ```
 
+## Conditionalizing compilation for Embedded Swift
+
+It's often useful to have source code be compilable under both regular Swift and Embedded Swift. The following syntax is available for that (but note that as the rest of Embedded Swift, it's experimental, subject to change and not considered source stable):
+
+```swift
+func sayHello() {
+  #if hasFeature(Embedded)
```

Comment:
`$Embedded`; `hasFeature` doesn't check if the feature is enabled if I recall correctly

Reply:
This is an important distinction we should call out because user guidance is to use `hasFeature()`, not `$`.

Or maybe there should be a `targetEnvironment(embedded)` condition.